pkgs/applications/misc/alacritty/default.nix

```diff
@@ -31,25 +43,19 @@ buildRustPackage rec {
   buildInputs = [
     cmake
     makeWrapper
-    freetype
-    fontconfig
     xclip
     pkgconfig
-    expat
-    libX11
-    libXcursor
-    libXxf86vm
-    libXi
-  ];
+  ] ++ rpathLibs;
 
   installPhase = ''
     mkdir -p $out/bin
     for f in $(find target/release -maxdepth 1 -type f); do
       cp $f $out/bin
     done;
-    wrapProgram $out/bin/alacritty --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath buildInputs}"
+    patchelf --set-rpath "${stdenv.lib.makeLibraryPath rpathLibs}" $out/bin/alacritty
   '';
 
+  dontPatchELF = true;
```

Comment:
If we go this route the platform should also exclude `darwin`. The current build fails anyway: https://hydra.nixos.org/eval/1378396?filter=alacritty&compare=1378360&full=